tests/v2/validation/provisioning/airgap/rke2_custom_cluster_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build provisioning.airgap || validation
```

Comment:
general question; non-provisioning tags have resolution down to engine type (rke1, rke2k3s). Is this possible / do you want that for provisioning tests in airgap and hosted sections? if not, is there a reason we have it for non-provisioning tests (i.e. scaling.gke)?

Reply:
I mean for consistency sake. I'll make the update then, good catch. I did this for the other provisioning tests.

Reply:
updated